packages/messageProxy/src/doc/doc.service.ts

```diff
@@ -8,12 +8,23 @@ import {
     JoinResponsePayload,
     AwarenessPayload,
 } from '../messages';
-import { BroadcastMessage, PubSub } from '../pubsub/types';
-import { WebSocketClient } from '../ws/types';
-import { KafkaPubSubToken } from '../pubsub/kafka-pubsub.service';
+
+import {
+    AwarenessBroadcastMessage,
+    awarenessBroadcastMessageType,
+    ChangesBroadcastMessage,
+    changesBroadcastMessageType,
+    DetachDocBroadcastMessage,
+    detachDocBroadcastMessageType,
+    DisconnectBroadcastMessage,
+    disconnectBroadcastMessageType,
+} from '../pubsub/types';
+import { WebSocketConnection } from '../ws/types';
 import { ActorService } from '../actor/actor.service';
 import { DocKey } from './types';
 import { NotJoinedError } from '../errors';
+import { PubsubService } from 'src/pubsub/pubsub.service';
+import { AttachDocBroadcastMessage, attachDocBroadcastMessageType } from 'src/pubsub/types/attachDocMessage';
```

Comment:
- same about `src/`
- add `index.ts` in pubsub and export everything in it to be able `import { PubsubService, AttachDocBroadcastMessage, attachDocBroadcastMessageType} from '../pubsub';`